src/operations/group.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
```

Comment:
There's a lot of `any` left in here, and its kinda ugly to type without _bsontype existing on Code. I figured since this is deprecated API it was something we would be okay with leave untyped especially since its internal at this layer anyway.
This might actually go away actually with the annotations work